lib/scrabble.rb

```diff
@@ -1,4 +1,11 @@
 class Scrabble
+
+  attr_reader :score
+
+  def initialize
```

Comment:
You don't _need_ this initialize method - you can access the `score` method you created without the attr_reader :)